src/components/modelConfig/ModelConfig.jsx

```diff
@@ -19,9 +19,9 @@ const renderSelectModel = (provider, handleModelChange) => {
     );
 }
 
-export const ModelConfig = ({ provider, handleProviderChange, model, handleModelChange, APIKey, handleAPIKeyChange, handleModelConfigSave }) => {
+export const ModelConfig = ({ provider, handleProviderChange, handleModelChange, APIKey, handleAPIKeyChange, handleModelConfigSave }) => {
     const needAPIKey = provider == null ? true : providerModels[provider].needAPIKey;
-    const enableSave = provider != null && model != null && (!needAPIKey || APIKey != '') ? {} : { disabled: true };
+    const enableSave = provider != null && (!needAPIKey || APIKey != '') ? {} : { disabled: true };
```

Comment:
this is bit hard to read... maybe simplify to `const disabled = provider == null || (needAPIKey && APIKey === '');`

Reply:
and we can just use the param `disabled={disabled}` as the param on `<button>`